packages/assets-controllers/src/CurrencyRateController.ts

```diff
@@ -90,6 +90,8 @@ export class CurrencyRateController extends BaseControllerV2<
 
   private includeUsdRate;
 
+  #enabled; // A boolean that controls whether or not network requests can be made by the controller.
```

Comment:
Nit: Instead of adding documentation with an inline comment what do you think about using JSDoc? You should be able to say:

``` suggestion
  /**
   * A boolean that controls whether or not network requests can be made by the controller
   */
  #enabled;
```

and then I believe you should see this when hovering over `this.#enabled`.